src/example-types/metricGroups/tagsQueryGroupStats.js

```diff
@@ -0,0 +1,9 @@
+let { groupStats } = require('./groupStatUtils')
+let { buildResultQuery, filter } = require('../filters/tagsQuery')
+
+// node = { field, tags, join, exact }
```

Comment:
This is just letting you know what properties this would expect on the grouping type which should match tagsQuery so you can just use its whole UI out of the box